### PR TITLE
Add "web" group for repos used by web applications

### LINKED ISSUE #5: Create group for front-end repos
This would be useful for front-end developers.

### PR DESCRIPTION
**Pull request by dmfalke**
@jtlong3rd can you take a look at this and make sure I'm not leaving out any repo?

@rbelnap do you have reservations with the group simply being named "web"?

### CODE REVIEW

**Review by jtlong3rd — APPROVED** [review]
Looks great! I guess we'll want to include CoreUI as well?

File: manifest.yml [hunk 1]
```diff
@@ -454,3 +486,21 @@ groups:
...
+      - web-study-data-access
+      - web-user-datasets
+      - web-multi-blast
+      - web-app-preferred-organisms
+      - WDKClient
+      - EbrcWebsiteCommon
+      - ApiCommonWebsite
+      - ClinEpiWebsite
+      - MicrobiomeWebsite
+      - OrthoMCLWebsite
```
Comment (jtlong3rd) [inline manifest.yml]:
```suggestion
      - OrthoMCLClient
```

**Comment by dmfalke** [comment]
@rbelnap would you mind reviewing this? Thanks!

**Review by rbelnap — APPROVED** [review]
LGTM

**Comment by dmfalke** [comment]
> LGTM

Thanks!

**Pull request merged**